docs/data-sources/elb_active_standby_pools.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: Dedicated Load Balance (Dedicated ELB)
+---
+
+# huaweicloud_elb_active_standby_pools
+
+Use this data source to get the list of active standby ELB pools.
+
+## Example Usage
+
+```hcl
+variable "pool_name" {}
+
+data "huaweicloud_elb_active_standby_pools" "test" {
+  name = var.pool_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `name` - (Optional, String) Specifies the name of the active/standby backend cloud server group.
+
+* `pool_id` - (Optional, String) Specifies the ID of the active/standby backend server group.
+
+* `description` - (Optional, String) Specifies supplementary information about the active/standby backend server group.
+
+* `loadbalancer_id` - (Optional, String) Specifies the ID of the load balancer with which the active/standby backend
+  server group is associated.
+
+* `healthmonitor_id` - (Optional, String) Specifies the ID of the health check configured for the active/standby backend
+  server group.
+
+* `protocol` - (Optional, String) Specifies the protocol used by the active/standby backend server group to receive
+  requests from the load balancer. Value options: **TCP**, **UDP**, **HTTP**, **HTTPS** or **QUIC**.
+
+* `member_address` - (Optional, String) Specifies the private IP address bound to the active/standby backend server.
+  This parameter is used only as a query condition and is not included in the response.
+
+* `member_device_id` - (Optional, String) Specifies the ID of the cloud server that serves as a active/standby backend
+  server. This parameter is used only as a query condition and is not included in the response.
+
+* `member_instance_id` - (Optional, String) Specifies the active/standby backend server ID. This parameter is used only
+  as a query condition and is not included in the response.
+
+* `listener_id` - (Optional, String) Specifies the ID of the listener to which the forwarding policy is added.
+
+* `type` - (Optional, String) Specifies the type of the active/standby backend server group. Value options:
+  **instance**, **ip** or **""**.
```

Comment:
what did you mean for the empty string?
In terraform, if the parameter is left blank, it means an empty string.
The enumerated value does not need to be specified.
If necessary, the following format is recommended:
```
* `type` - (Optional, String) Specifies the type of the active/standby backend server group.
  The valid values are as follows:
  + **instance**: Any type of backend servers can be added. If this value is specified, the `vpc_id` is mandatory.
  + **ip**:  Only IP as backend servers can be added. If this value is specified, the `vpc_id` cannot be specified.

  If the `type` is empty, any type of backend servers can be added.
```

Reply:
done